fenix/app/src/main/java/org/mozilla/fenix/trackingprotection/ProtectionsStore.kt

```diff
@@ -74,22 +92,27 @@ sealed class ProtectionsAction : Action {
  * @property url Current URL to display
  * @property isTrackingProtectionEnabled Current status of tracking protection for this session
  * (ie is an exception)
- * @property isCookieBannerHandlingEnabled Current status of cookie banner handling protection
+ * @property cookieBannerUIMode Current status of cookie banner handling protection
  * for this session (ie is an exception).
  * @property listTrackers Current Tracker Log list of blocked and loaded tracker categories
  * @property mode Current Mode of TrackingProtection
  * @property lastAccessedCategory Remembers the last accessed details category, used to move
  * accessibly focus after returning from details_mode
+ * @property isSnackBarVisible Current status of snackBar after the user reports a site
+ * domain where the cookie banner reducer didn't work.
  */
 data class ProtectionsState(
     val tab: SessionState?,
     val url: String,
     val isTrackingProtectionEnabled: Boolean,
-    val isCookieBannerHandlingEnabled: Boolean,
+    val cookieBannerUIMode: CookieBannerUIMode,
     val listTrackers: List<TrackerLog>,
     val mode: Mode,
     val lastAccessedCategory: String,
+    val siteDomainToReport: String,
+    val isSnackBarVisible: Boolean,
```

Comment:
As these are part of cookie banner maybe we could add them on `cookieBannerUIMode`, making it a seal class.

Reply:
val isSnackBarVisible: Boolean is not needed any more , I deleted it.